srv/SetSimulationPaused.srv

```diff
@@ -0,0 +1,9 @@
+# Pauses or unpauses the simulation
+
+bool pause                          # If true, sets simulation state to paused, otherwise sets it to running.
+                                    # If already in target state, nothing happens but status_message informs about it.
+
+---
+
+bool success                        # Return true if pausing / resuming was successful.
```

Comment:
This is an example of where a ternary response would make a lot more sense. Is the change successful, was it a no-op, or did it fail to pause. Versus the binary success.